app/mailer_previews/discussion_mailer_preview.rb

```diff
@@ -0,0 +1,25 @@
+# Rails Email Preview controller for previewing Discussion-related emails
+class DiscussionMailerPreview
+  def parent_comment_notification
+    DiscussionMailer.parent_comment_notification fetch_comment
+  end
+
+  def discussion_notification
+    DiscussionMailer.discussion_notification fetch_comment
+  end
+
+  def overview_notification
+    DiscussionMailer.overview_notification fetch_comment
+  end
+
+  private
+
+  def fetch_comment
+    @comment_id ? Comment.find( @comment_id ) : mock_comment
+  end
+
+  def mock_comment
+    # TODO: Use factory?
```

Comment:
For now I'd try to avoid bringing in the extra overhead (although probably small) of loading the factorybot + faker gems into production just for this.

It also avoids the risk of calling a factory that unintentionally persists some data to the production db.

E.g. Calling `FactoryBot.build_stubbed(:comment)` or even just `FactoryBot.attributes_for(:comment)` will usually be harmless, but its possible the comments factory gets configured to save a User or something else to the db on these calls, so saving test data in the prod. db.